configs/default/Add BEFH-BETAFPVF411.config

```diff
@@ -0,0 +1,111 @@
+# Betaflight / STM32F411 (S411) 4.1.1 Nov 15 2019 / 12:55:17 (1e5e3d369) MSP API: 1.42
```

Comment:
The name of the file is wrong now, please rename and remove the 'Add '.

Reply:
I had removed the Add

Reply:
Still there, try again: 
![image](https://user-images.githubusercontent.com/4742747/70514550-bf7f6e80-1b98-11ea-9f35-1d92680771a4.png)

Reply:
I had removed the Add again

Reply:
![image](https://user-images.githubusercontent.com/4742747/70519154-5996e500-1ba0-11ea-960f-fcd4de69050c.png)
Nope, still there.

Reply:
![image](https://user-images.githubusercontent.com/58228195/70525581-a966b380-1b82-11ea-8fd7-c3f8b3039713.png)

Reply:
Now is it right?

Reply:
No. You need to change the file name, not the title of the pull request:

![image](https://user-images.githubusercontent.com/4742747/70567074-1b2e1400-1bfa-11ea-90b7-21ca2e23fc17.png)

Reply:
![image](https://user-images.githubusercontent.com/58228195/70601254-9e685d80-1c2c-11ea-90ae-9101dc496092.png)

Reply:
I should hou to delete the same name's file?